public/index.html

```diff
@@ -24,7 +24,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <link
```

Comment:
Not necessary to add bootstrap CDN again, since you've already added it as a dependency to the project. See package.json.